docker-compose-rule-core/src/main/java/com/palantir/docker/compose/configuration/ShutdownStrategy.java

```diff
@@ -47,14 +29,16 @@
      */
     ShutdownStrategy KILL_DOWN = new KillDownShutdownStrategy();
     /**
-     * Skip shutdown, leaving containers running after tests finish executing.
+     * Skip down, leaving containers running after tests finish executing.
```

Comment:
intentional?

Reply:
Yes, "down" is a docker-compose term and is more clear than "shutdown" which is not a docker-compose term.